homeassistant/components/rfxtrx/__init__.py

```diff
@@ -328,7 +328,7 @@ def __init__(self, name, event, datas, signal_repetitions):
         self._event = event
         self._state = datas[ATTR_STATE]
         self._should_fire_event = datas[ATTR_FIRE_EVENT]
-        self._unique_id = f"{slugify(self._event.device.type_string.lower())}_{slugify(self._event.device.id_string.lower())}"
+        self._unique_id = f"{event.device.packettype:x}_{event.device.subtype:x}_{slugify(event.device.id_string.lower())}"
```

Comment:
Why even slugify and lower case it? why not just use `id_string` as-is ?

Reply:
Just assumed it needed to be a clean string. There can be semicolons and such in it.

Reply:
Home Assistant just sees it as a string. We never show it to the user.

Reply:
Dropped slugify